src/ui/RichText.tsx

```diff
@@ -129,7 +137,12 @@ export type RichTextProps<Blocks extends Schema> = {
   text?: ComponentType<{children: string}>
 } & {
   [K in keyof typeof RichTextElements]?:
-    | ComponentType<JSX.IntrinsicElements[K]>
+    | ComponentType<
+        React.JSX.IntrinsicElements[Extract<
```

Comment:
This change is not needed if we use html tag names in RichTextElements

Reply:
Is resolved.